### PR TITLE
Added Submodule Init to Workflow

### PR DESCRIPTION
**Pull request by will-rice**
I believe this is the only discrepancy remaining.

### CODE REVIEW

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -4,27 +4,28 @@
 name: Upload Python Package
 
 on:
-    release:
```
Comment (brentspell) [inline .github/workflows/python-publish.yml:7]: See if you can figure out a way to make this change without reformatting the whole file. If you have to reformat it, do that on a separate commit.

Comment (will-rice) [reply .github/workflows/python-publish.yml:7]: I believe I fixed it. it was pre-commit.

**Comment by brentspell** [comment]
nice, lgtm
